open-api.yaml

```diff
@@ -827,6 +827,11 @@ components:
     distinctAttribute:
       type: string
       description: Search returns documents with distinct (different) values of the given field.
+      default: 1500
+      nullable: false
```

Comment:
Not sure about the nullable field being useful on the schema description

Reply:
@ManyTheFish and @Kerollmops 
`null` is the value by default, not 1500.

Internally `null` means `1500`, but not for the API. It's internal, to avoid breaking changes